composer.json

```diff
@@ -13,11 +13,11 @@
     ],
     "require": {
         "php": ">=5.3.0",
```

Comment:
Since this modification seems to break BC, maybe you should bump to `"php": "^5.6||^7.0"`? 
It's not ideal to have an open upper constraint...

Reply:
You are correct, good one!

However I have to make this `^5.4||^7.0` since Laravel 5 works on PHP 5.4.